junit-4/src/test/resources/conf/parallel.conf.json

```diff
@@ -22,6 +22,12 @@
     },
     {
       "browser": "Internet Explorer"
+    },
```

Comment:
please confirm if bstackdemo works on IE across win versions. Similar issue faced somewhere else

Reply:
this works for the [test case](https://www.browserstack.com/docs/automate/selenium/getting-started/java#run-your-first-test) mentioned in the docs. Here is a [session](https://automate.browserstack.com/builds/4e2971b089be29ad170e5c0b0cedf75269134f3c/sessions/a5368388ae1e4c8370a2647e226136c16ac89352?auth_token=0673386fed96ba7e02373005e1bdafb2babc52e9f65d69742569a8b5890d63f4) on IE.